src/bci_build/package/__init__.py

```diff
@@ -31,7 +31,11 @@
 
 #: Remove various log files. While it is possible to just ``rm -rf /var/log/*``,
 #: that would also remove some package owned directories (not %ghost)
-LOG_CLEAN: str = "rm -rf /var/log/{lastlog,tallylog,zypper.log,zypp/history,YaST2}"
+LOG_CLEAN: str = """
+# Remove various log files. While it's possible to just rm -rf /var/log/*
+# would also remove some package owned directories (not %ghost)
+# and some files are actually wanted
+rm -rf /var/log/{lastlog,tallylog,zypper.log,zypp/history,YaST2}"""
```

Comment:
You cannot do this, `LOG_CLEAN` must be a single line instruction, because it is used in all `Dockerfile` templates inside of `RUN` instructions.

Reply:
restored